homeassistant/setup.py

```diff
@@ -240,12 +258,26 @@ def log_error(msg: str) -> None:
     elif name in processed:
         return
 
-    if hasattr(module, 'DEPENDENCIES'):
-        dep_success = await _async_process_dependencies(
-            hass, config, name, module.DEPENDENCIES)  # type: ignore
+    manifest = await hass.async_add_executor_job(
```

Comment:
@balloob Something about loading this manifest causes a lot of MQTT tests to fail. After about 90 minutes of investigation and debugging tonight, I was unable to figure out what's wrong with it (and I'm not familiar enough with the MQTT tests here). Would you be able to take a look and advise please?